src/core_plugins/kibana/public/management/sections/settings/lib/get_aria_name.js

```diff
@@ -0,0 +1,13 @@
+
+import { words } from 'lodash';
+
+/**
+ * @name {string} the name of the configuration object
+ * @returns {string} a space demimited, lowercase string with
+ *          special characters removed.
+ *
+ * Example: 'xPack:fooBar:foo_bar_baz' -> 'x pack foo bar foo bar baz'
```

Comment:
I don't think there is a master list. However, if there are specific cases you think we should test for, I can add additional tests. Seems overkill to process / maintain the entire list, when the logic really boils down to how it handles certain characters.